dashboard/lib/queries/lti.rb

```diff
@@ -2,9 +2,8 @@
 require 'authentication_option'
 
 class Queries::Lti
-  def self.get_user(id_token)
-    auth_id = Policies::Lti.generate_auth_id(id_token)
-    User.find_by_credential(type: AuthenticationOption::LTI_V1, id: auth_id)
+  def self.get_user(user_id)
+    LtiUserIdentity.find_by(subject: user_id)&.user
```

Comment:
we can find a user in the `LtiUserIdentity` table by its `user_id` so I removed the `id_token` as parameter in favor of just the `user_id`